features/build/stibuild.feature

```diff
@@ -90,3 +90,41 @@ Feature: stibuild.feature
     Then the output should not contain:
       | save-artifacts: No such file or directory|
 
+  # @author wzheng@redhat.com
+  # @case_id OCP-11120
+  Scenario: STI build with dockerImage with specified tag
+    Given I have a project
+    When I run the :new_app client command with:
+      | docker_image | centos/ruby-22-centos7                  |
+      | app_repo     | https://github.com/openshift-qe/ruby-ex |
+    Then the step should succeed
+    And the "ruby-ex-1" build completes
+    When I run the :patch client command with:
+      | resource      | buildconfig                                                                                                                                      |
+      | resource_name | ruby-ex                                                                                                                                          |
+      | p             | {"spec": {"strategy": {"sourceStrategy": {"from": {"kind": "DockerImage","name": "docker.io/centos/ruby-22-centos7:latest"}}},"type": "Source"}} |
+    Then the step should succeed
+    When I run the :start_build client command with:
+      | buildconfig | ruby-ex |
+    Then the step should succeed
+    And the "ruby-ex-2" build completes
+    Then the step should succeed
+    When I run the :patch client command with:
+      | resource      | buildconfig                                                                                                                                     |
+      | resource_name | ruby-ex                                                                                                                                         |
+      | p             | {"spec": {"strategy": {"sourceStrategy": {"from": {"kind": "DockerImage","name": "docker.io/centos/ruby-22-centos7:error"}}},"type": "Source"}} |
+    Then the step should succeed
+    When I run the :describe client command with:
+      | resource | buildconfig |
+      | name     | ruby-ex     |
+    Then the step should succeed
```

Comment:
This step doesn't seem to help anyhow. What is the purpose?

If purpose is to make sure `patch` succeeded, then you can use step
```
/^(?:(as admin) )?I successfully#{OPT_WORD} patch resource "(.*)\/(.*)" with:$/
```

Reply:
Yes, these steps should be deleted, thanks for reminder : )